WcaOnRails/app/views/devise/registrations/new.html.erb

```diff
@@ -2,6 +2,8 @@
   <%= bootstrap_devise_error_messages! %>
   <div class="panel panel-default">
     <div class="panel-heading">
+      <%# FIXME not sure the default is needed anymore : locale 'en' file is %>
+      <%# present for devise, and default locale fallback to 'en'. %>
```

Comment:
Yeah, I bet this can go. This is leftover from when I copied these files out of devise for our own modifications.